.github/workflows/release.yml

```diff
@@ -66,70 +72,7 @@ jobs:
 
       - uses: actions/upload-artifact@v4
         with:
-          name: "ubuntu-latest-${{ matrix.python-version[1] }}"
-          path: connectorx-python/target/wheels/*.whl
-
-  linux-aarch:
-    runs-on: ubuntu-22.04-arm
-    strategy:
-      matrix:
-        python-version: ["3.10", "3.11", "3.12", "3.13"]
-    steps:
-      - name: Checkout Repository
-        uses: actions/checkout@v5
-
-      # Check architecture of the target machine
-      - name: Check Target Architecture
-        run: |
-          echo "Architecture: $(uname -m)"
-          if [[ $(uname -m) != "aarch64" ]]; then
-            echo "Error: This workflow requires ARM architecture (aarch64)."
-            exit 1
-          fi
-
-      - name: Setup Python
-        uses: actions/setup-python@v5
-        with:
-          python-version: ${{ matrix.python-version }}
-
-      - name: Setup Rust
-        uses: actions-rs/toolchain@v1
-        with:
-          toolchain: stable
-          components: rustfmt
-          target: aarch64-unknown-linux-gnu
-          default: true
-
-      - name: Setup Just
-        uses: extractions/setup-just@v3
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Install Tools
-        run: |
-          sudo apt-get update -y
-          sudo apt-get install -y mysql-client libpq-dev freetds-dev krb5-locales clang libkrb5-dev build-essential gcc musl-tools libsasl2-modules-gssapi-mit libgssapi-krb5-2 krb5-user
-      - name: Setup Poetry
-        uses: Gr1N/setup-poetry@v9
-
-      - name: Setup Project
-        run: |
-          just bootstrap-python
-          python -m pip install --upgrade pip
-          pip install maturin
-
-      - name: Build Wheel (Native)
-        run: |
-          maturin build -m connectorx-python/Cargo.toml --target aarch64-unknown-linux-gnu -i python${{ matrix.python-version }} --release
-        env:
-          SQLITE3_STATIC: 1
-          KRB5_INCLUDE_DIR: /usr/include
-          KRB5_LIB_DIR: /usr/lib/aarch64-linux-gnu
-
-      - name: Upload Artifact
-        uses: actions/upload-artifact@v4
-        with:
-          name: "linux-arm-${{ matrix.python-version }}"
+          name: "linux-${{matrix.platform.arch}}-${{ matrix.python-version[1] }}"
```

Comment:
Since the name is changed from `ubuntu-latest-xxx` to `linux-x86_64-xxx`. The named referred in the `verify` task should also be updated. Otherwise it cannot find the artifact: https://github.com/davidhewitt/connector-x/actions/runs/18595020541/job/53024260726#step:4:16